R/plot_shapley.R

```diff
@@ -58,8 +58,9 @@ plot.shapley.multipleValues = function(shap.values) {
 #'
 #' @description Orders the values by their sign and value, shifts them and returns
 #'   them as a vector.
-#' @param points shapley.values
+#' @param points shapley.values. You can get those with getShapleyValues(shapley())
 #' @param shift data.mean
+#' @export

```

Comment:
No this function is not mentioned to be exported! This one is just for internal use